packages/bot-skeleton/src/scratch/blocks/Advanced/Variable/variables_get.js

```diff
@@ -3,6 +3,16 @@ import { localize } from '@deriv/translations';
 Blockly.Blocks.variables_get = {
     init() {
         this.jsonInit(this.definition());
+        this.inputList.forEach(input_list => {
+            input_list.fieldRow.forEach(fieldRow => {
+                setTimeout(() => {
+                    if (fieldRow?.borderRect_ || fieldRow.clickTarget_) {
+                        const target_block = fieldRow?.borderRect_ || fieldRow.clickTarget_;
+                        Blockly.utils.dom.addClass(target_block, 'blocklyVariableGet');
+                    }
+                }, 0);
```

Comment:
```suggestion
            input_list.fieldRow.forEach(fieldRow => {
                setTimeout(() => {
                    if (fieldRow?.borderRect_ || fieldRow.clickTarget_) {
                        const target_block = fieldRow?.borderRect_ || fieldRow.clickTarget_;
                        Blockly.utils.dom.addClass(target_block, 'blocklyVariableGet');
                    }
                }, 0);
```
create a promise here and use map to put it in an array and after that

```
Promise.all(promiseArray);
```

Reply:
removed this foreach loop and handled with css